### PR TITLE
Add centrifuge kreport in taxprofiler

### PR DESCRIPTION
**Pull request by sofstam**
This PR adds [centrifuge_kreport](https://github.com/nf-core/modules/tree/master/modules/centrifuge/kreport) in taxprofiler . It creates Kraken-style reports from centrifuge out files.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -254,6 +254,16 @@ process {
         ext.prefix = params.perform_runmerging ? { "${meta.id}-${meta.db_name}" } : { "${meta.id}-${meta.run_accession}-${meta.db_name}" }
     }
 
+    withName: CENTRIFUGE_KREPORT {
+        ext.args = { "${meta.db_params}" }
+        ext.prefix = { "${meta.id}" }
```
Comment (jfy133) [inline conf/modules.config:259]:
```suggestion
        ext.prefix = params.perform_runmerging ? { "${meta.id}-${meta.db_name}" } : { "${meta.id}-${meta.run_accession}-${meta.db_name}" }
```

File: nextflow.config [hunk 2]
```diff
@@ -98,6 +98,7 @@ params {
 
     // centrifuge
     run_centrifuge             = false
+    centrifuge_kreport         = false
```
Comment (jfy133) [inline nextflow.config:101]:
```suggestion
```

Not necessary, I think this should run by default if centrifuge is executed (it should be a small enough process I think)

**Review by jfy133 — APPROVED** [review]
Once prettier passes I think it overall looks good!

Just need to add our 'standard' prefix, and also I suggest to remove the flag from nextflow.config you don't actually use (I agree it makes more sense it should execute by default with centrifiuge)

**Review by jfy133 — APPROVED** [review]
LGTM now!

**Comment by sofstam** [comment]
Great, I will merge!

**Pull request merged**